proguard.txt

```diff
@@ -1,10 +1,11 @@
--dontobfuscate
+-keeppackagenames com.bugsnag.**
+-keepnames class com.bugsnag.**
+-keepclassmembernames class com.bugsnag.** { *; }
 
--renamesourcefileattribute SourceFile
 -keepattributes Exceptions,InnerClasses,Signature,Deprecated,
                 SourceFile,LineNumberTable,*Annotation*,EnclosingMethod
 
--keep public class * {
+-keep public class com.bugsnag.** {
```

Comment:
@kattrali I wonder if the rule is needed at all when proguard runs during the app-build instead of during the library-build